Makefile

```diff
@@ -15,22 +15,28 @@ venv:
 	venv/bin/pip install tox
 	venv/bin/pip install -e .
 
-test-integration:
-	podman build --no-cache -t cachi2-${USER} .
-	CACHI2_IMAGE=localhost/cachi2-${USER}:latest tox -e integration
-
 test:
-	PATH="${PWD}/venv/bin:${PATH}" tox
+	venv/bin/tox
 
 test-unit:
-	PATH="${PWD}/venv/bin:${PATH}" tox -e $(TOX_ENVLIST) -- $(TOX_ARGS)
+	venv/bin/tox -e $(TOX_ENVLIST) -- $(TOX_ARGS)
+
+test-integration:
+	venv/bin/tox -e integration
```

Comment:
With this change here, the tests will always build and use a `cachi2:inttest` image, whereas the `make build-image` command will build a `cachi2:latest` image. Was that the intended effect? It feels to me that the build-image[-pristine] commands are not very useful now.

Reply:
All in all, I like to have the image rebuilt (with cache) and the integration tests run with a single `make test-integration` command.

Reply:
Yeah, that's true. I felt like someone could be surprised if running integration tests rebuilt their `cachi2:latest` image, but looking back at it, I think that's reasonable behavior

Reply:
Updated to make int. tests build cachi2:latest by default